docs/client-libs/dart.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Dart Client Library
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+# Dart Client Library
```

Comment:
Please add references to repositories/generated documentation for client libraries here and on Java and JS pages as well.

Reply:
It's a good idea! Added.